server/pdf.js

```diff
@@ -0,0 +1,14 @@
+const markdown = require('markdown').markdown
+const pdf = require('html-pdf')
+
+const servePDF = (text, response) => {
+  const parsedHTML = markdown.toHTML(text)
+  pdf.create(parsedHTML).toStream((err, stream) => {
```

Comment:
mikä tää Stream on, miten se toimii?

Reply:
https://medium.freecodecamp.org/node-js-streams-everything-you-need-to-know-c9141306be93